app/controllers/account_follow_controller.rb

```diff
@@ -6,7 +6,7 @@ class AccountFollowController < ApplicationController
   before_action :authenticate_user!
 
   def create
-    FollowService.new.call(current_user.account, @account.acct)
+    FollowService.new.call(current_user.account, @account.acct) unless @account.stealth_blocking?(current_user.account)
```

Comment:
Would this not be more fool-proof to be within `FollowService`?

Reply:
I guess it would. The idea was that `FollowService` currently either errors out or return a `Follow` or `FollowRequest` object, so making it not return one while not erroring out would introduce a new behavior.
I guess I could change that.